src/Strategy/Haskell/Stack.hs

```diff
@@ -11,22 +11,74 @@ module Strategy.Haskell.Stack (
 ) where
 
 import App.Fossa.Analyze.Types (AnalyzeProject, analyzeProject)
-import Control.Effect.Diagnostics
+import Control.Effect.Diagnostics (
+  Diagnostics,
+  Has,
+  context,
+  fromEither,
+ )
 import Control.Monad (when)
-import Data.Aeson.Types
+import Data.Aeson.Types (
+  FromJSON (parseJSON),
+  ToJSON,
+  withObject,
+  (.!=),
+  (.:),
+  (.:?),
+ )
 import Data.Foldable (for_)
 import Data.Map.Strict qualified as Map
 import Data.String.Conversion (toString)
 import Data.Text (Text)
-import Discovery.Walk
-import Effect.Exec
-import Effect.Grapher
+import Discovery.Walk (
+  WalkStep (WalkContinue, WalkSkipAll),
+  findFileNamed,
+  walk',
+ )
+import Effect.Exec (
+  AllowErr (Never),
+  Command (Command, cmdAllowErr, cmdArgs, cmdName),
+  Exec,
+  execJson,
+ )
+import Effect.Grapher (
+  MappedGrapher,
+  direct,
+  edge,
+  mapping,
+  withMapping,
+ )
 import Effect.ReadFS (ReadFS)
 import GHC.Generics (Generic)
 import Graphing qualified as G
-import Path
-import Types
-import Prelude
+import Path (Abs, Dir, File, Path)
+import Types (
+  DepType (HackageType),
+  Dependency (
+    Dependency,
+    dependencyEnvironments,
+    dependencyLocations,
+    dependencyName,
+    dependencyTags,
+    dependencyType,
+    dependencyVersion
```

Comment:
This type is EVERYWHERE in the codebase, and we have to manually expand it every time.